tests/MercadoPago/Client/Invoice/InvoiceClientUnitTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace MercadoPago\Client\Invoice;
+
+use MercadoPago\MercadoPagoConfig;
+use MercadoPago\Net\MPDefaultHttpClient;
+use MercadoPago\Client\Base\BaseClient;
+
+/**
+ * InvoiceClient unit tests.
+ */
+final class InvoiceClientUnitTest extends BaseClient
+{
+    public function testGetSuccess(): void
+    {
+        $filepath = '../../../Resources/Mocks/Response/Invoice/invoice_base.json';
+        $mock_http_request = $this->mockHttpRequest($filepath, 200);
+
+        $http_client = new MPDefaultHttpClient($mock_http_request);
+        MercadoPagoConfig::setHttpClient($http_client);
+
+        $client = new InvoiceClient();
+        $invoice_id = 6114264375;
+        $invoice = $client->get($invoice_id);
+
+        $this->assertEquals(200, $invoice->getResponse()->getStatusCode());
```

Comment:
```suggestion
        $this->assertSame(200, $invoice->getResponse()->getStatusCode());
```

Reply:
I didn't know `assertSame`, it does the `===` check, also checking types, decimals, etc. Good idea!